src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -684,7 +684,20 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 		this.layout(this.dimensions, { forceRevealActiveTab: true });
 	}
 
+	private previousSelectedCount = 0;
```

Comment:
It would be nice if the old selection count was part of the event that leads to this update. I guess one challenge is that here you want the filtered selection count for the multi-tab-row case?